app/views/service_providers/show.html.erb

```diff
@@ -108,29 +108,44 @@
 <hr>
 
 <% if service_provider.identity_protocol == 'saml' %>
-  <h2 class="margin-bottom-0"><label for="acs_url">Assertion Consumer Service URL:</label> <%= service_provider.acs_url %></h2>
+  <h2 class="margin-bottom-0">
```

Comment:
**question:** you sure this isn't worth turning into a partial? There's a lot of repetition. I also don't know if you need the `for` here since it isn't part of a form so you'd really just need to pass in the label text and the value.

Reply:
I kept most of the html alone rather and just added some styles.  It might look like a huge diff but it's not.  I typically only like to use partials if they are going to be reused though I know that isn't a universal opinion.  I feel like it just makes it more difficult to look at .erbs when the code is spread out.

Reply:
As for the `for`, I think they could probably be removed everywhere.